### PR TITLE
feat(learn): migrate instructions from adnode boilerplate

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39720

Boilerplate PR: https://github.com/freeCodeCamp/boilerplate-advancednode/pull/15

I tried as much as possible to not change unrelated content, but I also took the opportunity to change the instructions with previously discussed versioning additions, and formatting to use GitHub Flavoured MD.

Separate Issue: Husky needs to be configured to not format (search for) changes within backticks. If tests fail, this will be why. Also, might be worthwhile (especially once we migrate to mdx) to change the prettier/lint config to do the same.

### CODE REVIEW

**Review by moT01 — DISMISSED** [review]
LGTM 🎉

File: curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/how-to-put-a-profile-together.md [hunk 1]
```diff
@@ -6,41 +6,55 @@ forumTopicId: 301554
 ---
 
 ## Description
+
 <section id='description'>
 
-Now that we can ensure the user accessing the <em>/profile</em> is authenticated, we can use the information contained in 'req.user' on our page!
+Now that we can ensure the user accessing the `/profile` is authenticated, we can use the information contained in `req.user` on our page!
+
+Pass the object containing the variable `username` equaling `req.user.username` into the render method of the profile view. Then, go to your `profile.pug` view, and add the following line below the existing `h1` element, and at the same level of indentation:
```
Comment (RandellDawson) [inline curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/how-to-put-a-profile-together.md:14]:
```suggestion
Pass an object containing the property `username` and value of `req.user.username` as the second argument for the render method of the profile view. Then, go to your `profile.pug` view, and add the following line below the existing `h1` element, and at the same level of indentation:
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my one suggestion that seems to make the sentence more readable.

**Comment by Sky020** [comment]
@RandellDawson Thanks, for that. It is much clearer.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@moT01 I believe this PR combined with the boilerplate PR referenced in the PR body are good to go.  We can merge this and then merge the boilerplate once these changes hit production.

**Review by moT01 — APPROVED** [review]
👍 🎉 🚀 @Sky020

**Pull request merged**